### DISCUSSION TITLE
Is it possible to load the guitar pro tab in local path?

### DISCUSSION THREAD

**Discussion by zhouhao27** [CoderLine/alphaTab#734] [Q&A] [answered]
I'm wondering if it's possible to load the guitar pro tab in local path instead of loading from a url?

**Comment by Reiji777** [comment]
The demo on the project website has an open local file option.
And I believe the source code is below:

https://github.com/CoderLine/alphaTabWebsite/blob/adc61927f9383bcb83b602dc2fce66b0ea938fbf/src/alphatab-full.js#L502

**Comment by Danielku15** [comment, marked as answer]
See https://github.com/CoderLine/alphaTab/discussions/650
Please try to search for old issues and discussions.
